src/elements/frame_element.ts

```diff
@@ -35,6 +35,12 @@ export class FrameElement extends HTMLElement {
     this.delegate.disconnect()
   }
 
+  reload() {
+    const srcAttribute = this.src;
+    this.src = null;
+    this.src = srcAttribute;
```

Comment:
We've using ES6 here, so let's do:

```suggestion
    const { src } = this
    this.src = null
    this.src = src
```